keywords/MobileRestClient.py

```diff
@@ -1105,6 +1105,10 @@ def update_doc(self, url, db, doc_id, number_updates=1, attachment_name=None, ex
         auth_type = get_auth_type(auth)
         doc = self.get_doc(url, db, doc_id, auth)
         current_rev = doc["_rev"]
+        try:
+            doc["updates"]
```

Comment:
I'm not sure what is that you are trying to do over here. It seems dangling.

Reply:
this is not related to 1.4.1 . if you call this method without having updates field in document, it fails. It expects to create field before you call this method. I added this logic so that if there is an exception while accessing updates field, then it will create and won't fail when it tries to increment in next call.